tests/ui/format_args.rs

```diff
@@ -14,6 +14,7 @@ use std::panic::Location;
 
 struct Somewhere;
 
+#[allow(tostring_impl)]
```

Comment:
```suggestion
#[allow(clippy::tostring_impl)]
```
CI also failed because of this. Without the prefix it's looking for a rustc lint called that and complains about an unknown lint.
(Also needs to be changed in the other test files)

Reply:
fixed